src/classes/DMLManager.cls

```diff
@@ -126,20 +126,9 @@ public inherited sharing class DMLManager {
 		checkCRUDPermission(new Set<SObjectType>{mergeList[0].getSObjectType()},Operation.OP_DELETE);
 		
 		// If no errors have been thrown to this point, execute the merge operation.
-		Database.merge(masterObj, mergeList);
+		mergeAsSystem(masterObj, mergeList);
 	}
 	
-	private static void upsertCollection(List<SObject> objList){
```

Comment:
All of Mike's unit tests for merge pass a collection of objects to merge in of size greater than 1. Can you write a test similar to the ones that already exist that just tries to pass in a collection of size 1? And a test that passes in a single merge SObject instance (we re-wrap it in a List<SObject> on line 52) so I want to make sure that works too

(Said differently: I'm concerned that this code is still necessary)

Reply:
Added new test named, "mergeAsSystem_When_PassingASingleSObject_Expect_Success"